### PR TITLE
Unify connection pool config

### PR DESCRIPTION
**Pull request by dgaubert**
- [x] Create a new general configuration entry for PostgreSQL pool.
 - [x] Move "postgres" attributes inside Mapnik's renderer configuration to make it clear where that belongs.
 - [x] Remove geojson renderer configuration
 - [x] Use same pool configuration to create windshaft's renderers

### CODE REVIEW

File: config/environments/production.js.example [hunk 1]
```diff
@@ -115,26 +115,10 @@ var config = {
...
-         * database connections to be closed on renderer
-         * expiration (1 minute after last use).
-         * Setting to true (the default) would never
-         * close any connection for the server's lifetime
-         */
-        persist_connection: false,
-        simplify_geometries: true,
-        use_overviews: true, // use overviews to retrieve raster
-        max_size: 500
+        port: 6432
```
[the remark this answers is not in the document]
Comment (dgaubert) [reply config/environments/production.js.example]: I just did it for `development` examples. Do you want me to change it also for `staging` and `production` examples??

[the remark this answers is not in the document]
Comment (rochoa) [reply config/environments/production.js.example]: I would use the default PostgreSQL port, using the pgbouncer's default one is a bit weird if you don't have pgbouncer.

File: lib/cartodb/server_options.js [hunk 2]
```diff
@@ -118,7 +127,7 @@ module.exports = {
             // TODO: allow to specify in configuration
             srid: 3857
         },
-        datasource: global.environment.postgres,
+        datasource: rendererConfig.mapnik.postgis,
```
[the remark this answers is not in the document]
Comment (dgaubert) [reply lib/cartodb/server_options.js]: Ups! Good catch! :)

**Comment by dgaubert** [comment]
Ready for another CR round ;)

**Pull request merged**